kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1457,6 +1457,16 @@ object KyuubiConf {
       .transform(_.toLowerCase(Locale.ROOT))
       .createWithDefault("simple")
 
+  val OPERATION_RESULT_COMPRESSION_CODEC: ConfigEntry[String] =
+    buildConf("kyuubi.operation.result.compression.codec")
```

Comment:
As the config is used for Arrow-based results, shall we change it to `kyuubi.operation.result.arrow.compression`?